.github/workflows/automated-acceptance-tests.yml

```diff
@@ -20,43 +21,113 @@ env:
   AWS_DEFAULT_REGION: eu-central-1
 
 jobs:
-  build:
+  build_and_deploy:
     runs-on: ubuntu-latest
     environment: ${{ inputs.account }}
 
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Configure AWS credentials
+      - name: Assume Superwerker AWS Role
         uses: aws-actions/configure-aws-credentials@v4
         with:
           aws-region: ${{ env.AWS_DEFAULT_REGION }}
-          role-to-assume: ${{ secrets.AWS_RELEASE_ROLE }}
+          role-to-assume: ${{ secrets.AWS_AAC_NS_ROLE }}
```

Comment:
the role name is a bit misleading if we publish assets + add the NS records with the same one

Reply:
I agree. I will change it.